NOTICE

```diff
@@ -44,11 +44,13 @@ Amazon Technologies, Inc (http://www.amazon.com/).
 THIRD PARTY COMPONENTS
 **********************
 This software includes third party software subject to the following copyrights:
-- XML parsing and utility functions from TinyXml2 - Lee Thomason.
-- JSON parsing and utility functions from JsonCpp - Copyright (c) 2007-2010 Baptiste Lepilleur.
+- Very fast, header-only/compiled, C++ logging library from spdlog - Copyright (c) 2016 Gabi Melman
+- An open-source formatting library for C++ from fmt - Copyright (c) 2012 - present, Victor Zverovich
+- XML parsing and utility functions from TinyXml2 - Lee Thomason
+- JSON parsing and utility functions from JsonCpp - Copyright (c) 2007-2010 Baptiste Lepilleur
 - OpenSSL build files for cmake used for Android Builds - Copyright (C) 2007-2012 LuaDist and Copyright (C) 2013 Brian Sidebotham
 - Android tool chain cmake build files - Copyright (c) 2010-2011, Ethan Rublee and Copyright (c) 2011-2014, Andrey Kamaev
-- gsl-lite - Copyright (c) 2015 Martin Moene and Copyright (c) 2015 Microsoft Corporation. All rights reserved.
+- gsl-lite - Copyright (c) 2015 Martin Moene and Copyright (c) 2015 Microsoft Corporation. All rights reserved
```

Comment:
The dot was part of the original: https://github.com/gsl-lite/gsl-lite/blob/master/LICENSE#L4

Reply:
Ah, I did not know that these notices are literal. Will add it back again :)

Reply:
I don't know whether they need to be literal, but I tried to modify as little as possible while keeping then on a single line (for 1 3rd party)